hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.h

```diff
@@ -130,8 +131,8 @@ class ApiDbReader : public PartialOsmMapReader, public Boundable
   Tgs::BigMap<long, long> _relationIdMap;
   Tgs::BigMap<long, long> _wayIdMap;
 
-  geos::geom::Envelope _bounds;
-  geos::geom::Envelope _overrideBounds; // this will override _bounds
+  std::shared_ptr<geos::geom::Geometry> _bounds;
+  std::shared_ptr<geos::geom::Geometry>_overrideBounds; // this will override _bounds
```

Comment:
Just a nitpick, you removed a space between the template and the `_overrideBounds` name.

Reply:
Will fix. thanks